File: wallet/src/main/java/com/worldventures/wallet/ui/wizard/input/manual/WizardManualInputPresenter.java
package com.worldventures.wallet.ui.wizard.input.manual;

import com.worldventures.wallet.ui.common.base.WalletPresenter;

public interface WizardManualInputPresenter extends WalletPresenter<WizardManualInputScreen> {

   void goBack();

   void checkBarcode(String barcode);

   void retry(String barcode);

   void retryAssignedToCurrentDevice();
}

File: wallet/src/main/java/com/worldventures/wallet/ui/wizard/input/manual/WizardManualInputPresenter.kt
package com.worldventures.wallet.ui.wizard.input.manual

import com.worldventures.wallet.ui.common.base.WalletPresenter
import com.worldventures.wallet.ui.wizard.input.helper.BaseBarcodeInputPresenter

interface WizardManualInputPresenter : WalletPresenter<WizardManualInputScreen>, BaseBarcodeInputPresenter {

   fun goBack()

}
